docs/Users/Tutorials/Tutorial.rst

```diff
@@ -210,7 +210,7 @@ Lets add the following section to our ``.coafile``:
     [TODOS]
     bears = KeywordBear
 
-And execute coala with the ``-s`` argument which is the same as
+And execute ``coala`` with the ``-s`` argument which is the same as
```

Comment:
no, since here it is about the command, not the project, i think the double backticks fit better.